meshmode/discretization/__init__.py

```diff
@@ -1,4 +1,7 @@
-__copyright__ = "Copyright (C) 2013-2020 Andreas Kloeckner"
+__copyright__ = """
+Copyright (C) 2013-2021 Andreas Kloeckner
+Copyright (C) 2021 Thomas Gibson
```

Comment:
```suggestion
Copyright (C) 2021 University of Illinois Board of Trustees
```
https://otm.illinois.edu/disclose-protect/open-source-licensing

(Here and elsewhere)